test/snapshot.ts

```diff
@@ -84,8 +86,9 @@ describe('Snapshot', () => {
       assert(promisified);
     });
 
-    it('should localize parent.Promise', () => {
-      assert.strictEqual(snapshot.Promise, SUBSCRIPTION.Promise);
+    it('should localize parent.Promise', () => {  
+      snapshot = new Snapshot(PUBSUB,SNAPSHOT_NAME);
```

Comment:
to pass this test, please use a real `PubSub` object, 
```javascript
const pubsub = new PubSub();
snapshot = new Snapshot(pubsub, SNAPSHOT_NAME)
```

Reply:
done